api/helpers/handlebars-helpers.js

```diff
@@ -68,6 +68,9 @@ module.exports = {
     }
   },
   hasValue: (article, name) => {
```

Comment:
this function should return a boolean. i'm only using it [here](https://github.com/participedia/api/blob/master/views/partials/edit-richtext.html#L3) currently, but wanted to make it reusable in case it was needed for another field. if article is null for some reason, i think it would be preferable to return false rather than a string.